package.json

```diff
@@ -1,16 +1,5 @@
 {
   "author": "Wire Swiss <wireapp@wire.com>",
-  "contributors": [
-    {
-      "email": "chris@wire.com",
-      "name": "Chris Owen",
-      "url": "http://wire.com"
-    },
-    {
-      "email": "digital.flowers@hotmail.com",
-      "name": "Fareed Al-Namrouti"
-    }
-  ],
```

Comment:
This list was incomplete and can be found on https://github.com/wireapp/node-addressbook/graphs/contributors